Cargo.toml

```diff
@@ -139,7 +139,7 @@ fs-err = "2.9.0"
 fs2 = "0.4.3"
 futures = "0.3"
 futures-channel = "0.3"
-getrandom = { version = "0.2.7", features = ["custom"] }
+getrandom = "0.2.7"
```

Comment:
Why do we still depend on `getrandom`?

Reply:
See comment in `bindings/Cargo.toml` - to detect it if someone tries to use it. Also, we can't depend on the `std` feature of rand without pulling in getrandom, and getrandom will yell at you if you try to compile for wasm without enabling either `js` or `custom`. And if we did forgo `std`, every crate in a module's dependency tree would have to as well - they'd have to do `rand = { version  = "..", default-features = false }` which feels like some unnecessary friction, especially with the unhelpful advice from the `getrandom` crate telling you to enable the `js` feature.